src/OrchardCore.Modules/OrchardCore.Spatial/Manifest.cs

```diff
@@ -0,0 +1,11 @@
+using OrchardCore.Modules.Manifest;
+
+[assembly: Module(
+    Name = "Spatial",
+    Author = ManifestConstants.OrchardCoreTeam,
+    Website = ManifestConstants.OrchardCoreWebsite,
+    Version = ManifestConstants.OrchardCoreVersion,
+    Description = "This feature provides the ability to provide spatial locations to content items.",
+    Dependencies = new[] { "OrchardCore.ContentFields", "OrchardCore.Lucene" },
```

Comment:
why does it depend on `ContentFields` ? 

Probably the dependency should be `ContentTypes`

Reply:
Because it adds a Field to the ContentFields. And ContentFields is dependent on Content Types so it's all good.

Reply:
![image](https://user-images.githubusercontent.com/3228637/112653765-21fb8d00-8e25-11eb-8199-5fc4d9a98235.png)

Reply:
Or maybe what you thought/meant is that I could use the field independently of the ContentFields module. Need to try.

Reply:
Yup, that's what I thought / meant. Should be ok I think, the media field doesn't _need_ `ContentFields` to work ;)

Reply:
done